web/src/interfaces.ts

```diff
@@ -64,3 +64,22 @@ export interface Customer {
   address: string;
   numOngoingCommits: number;
 }
+
+/**
+ * MerchantPayload Interface that contains the fields of the payload that
+ * would be sent to the server to add the merchant to the database.
+ */
+export interface MerchantPayload {
+  name: string;
+  email: string;
+  vpa: string;
+}
+
+/**
+ * MerchantResponse Interface that contains the fields of the Response that
```

Comment:
Doc change: ...Interface that contains the fields of the response that client side would receive after merchant signup.

Reply:
It's not the response that client side would receive after merchant signup for now. But I changed it accordingly to make it clearer.